log/log.go

```diff
@@ -32,6 +32,11 @@ func Info(msg Message) {
 	global.printf(LevelInfo, msg, os.Stdout)
 }
 
+// Warning prints message in warning mode.
+func Warning(msg Message) {
+	global.printf(LevelWarning, msg, os.Stdout)
```

Comment:
I think we should output these logs into Stderr.

Reply:
done